### PR TITLE
Draft) Trying to fix JitPack builds.

### PR DESCRIPTION
**Pull request by DitFranXX**
https://github.com/inorichi/tachiyomi/issues/1894#issuecomment-475156483

It's draft, misopened.

### CODE REVIEW

File: gradle/publish-android-lib.gradle [hunk 1]
```diff
@@ -118,9 +119,7 @@ afterEvaluate { project ->
     task androidJavadocs(type: Javadoc) {
         source = android.sourceSets.main.java.srcDirs
 
-        classpath += project.files(android.getBootClasspath())
-        classpath += configurations.compileJavadoc
-        classpath += configurations.compile + configurations.provided
+        classpath += project.android.libraryVariants.toList().first().javaCompile.classpath
```
Comment (DitFranXX) [inline gradle/publish-android-lib.gradle]: When it uses `configuarations`, It fails to build becuase there's no `debug` or `release` things. (`failOnBuild false` doesn't work.)

Comment (inorichi) [reply gradle/publish-android-lib.gradle]: Maybe the entire task should be deleted? Although if it works like this, I don't mind.

**Comment by DitFranXX** [comment]
The commit 007f75e should fixes `compileClasspath` error.
I want to test it on JitPack ~~but currently Jitpack can't resolve (many?) dependencies: jitpack/jitpack.io#3787 jitpack/jitpack.io#3786 jitpack/jitpack.io#3782~~ It's Jcenter issue. thank you.

Currently `GROUP` is set as `com.github.inorichi.storio`

**Comment by inorichi** [comment]
Jcenter is having an outage atm https://www.reddit.com/r/androiddev/comments/b3olc4/jcenter_is_having_a_partial_outage_at_the_moment/

File: gradle/publish-android-lib.gradle (hunk 1 above)
Comment (DitFranXX) [reply gradle/publish-android-lib.gradle]: Done in 2e13fc2

**Comment by inorichi** [comment]
Merged the other PR.

**Pull request closed without merging**